pkg/api/validation/schema.go

```diff
@@ -123,11 +123,7 @@ func (s *SwaggerSchema) ValidateObject(obj interface{}, apiVersion, fieldName, t
 	for key, value := range fields {
 		details, ok := properties.At(key)
 		if !ok {
-			glog.Infof("unknown field: %s", key)
-			// Some properties can be missing because of
-			// https://github.com/GoogleCloudPlatform/kubernetes/issues/6842.
-			glog.Info("this may be a false alarm, see https://github.com/GoogleCloudPlatform/kubernetes/issues/6842")
-			continue
+			return append(allErrs, fmt.Errorf("found invalid field %s for %s", key, typeName))
```

Comment:
It's not the same case as line 103. We don't need to look at details.

Why can't we move on to the next field? That's what the code was doing before.